README.md

```diff
@@ -1,6 +1,14 @@
-# Multiplayer
-Multiplayer mod for RimWorld
+# Rimworld Multiplayer
+Originally created by Zetrith, Rimworld Multiplayer is now being updated and maintained by community members.
+
+Please do all pull requests to the development branch
+
+Join the Discord : https://discord.gg/S4bxXpv
 
 Thanks to Pardeike for making [Harmony](https://github.com/pardeike/Harmony) and RevenantX for creating [LiteNetLib](https://github.com/RevenantX/LiteNetLib)
 
-Please do all pull requests to the development branch
+[Download |](https://github.com/Parexy/Multiplayer/releases/download/v0.4.5/Multiplayer.zip)
```

Comment:
This link refers to the direct download of patch 0.4.5. What happens if we update the mod? This link should point to the release page.